spec/integration/account_management_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe "Account Management" do
+describe "Account Management", :skip do
```

Comment:
Why are we skipping these tests?

Reply:
The tests we are skipping are designed to rely on calling the `autheticate!` method [here](https://github.com/Sage/zuora/blob/a9af59e57518729c95f23e0c90c869005dc38e8b/lib/zuora/api.rb#L92), which then attempts to make a live network call with the correct credentials in order to return a proper auth object for the test cases we skipped. 

This was discussed during stand up last week whilst you were on vacation and we arrived at a conclusion to skip the affected tests because no one seem to have the correct credentials for the test purpose.